evm_arithmetization/src/cpu/kernel/constants/mod.rs

```diff
@@ -375,9 +377,12 @@ pub mod cancun_constants {
         hex!("000000000000000000000000000000001666c54b0a32529503432fcae0181b4bef79de09fc63671fda5ed1ba9bfa07899495346f3d7ac9cd23048ef30d0a154f"), // y_im
     ];
 
+    pub const BEACON_ROOTS_CONTRACT_ADDRESS: Address =
+        H160(hex!("000F3df6D732807Ef1319fB7B8bB8522d0Beac02"));
+
```

Comment:
To be fair, I think this should move to common crate instead (I think the initial PR on this topic just ignored the constants defined in `evm_arithmetization`).